docs/src/main/sphinx/develop/insert.rst

```diff
@@ -0,0 +1,30 @@
+=============================================
+Supporting ``INSERT`` and ``CREATE TABLE AS``
+=============================================
+
+To support ``INSERT``, a connector must implement:
+
+* ``beginInsert()`` and ``finishInsert()`` from the ``ConnectorMetadata``
+  interface;
+* a ``ConnectorPageSinkProvider`` that receives a table handle and returns
+  a ``ConnectorPageSink``.
+
+When executing an ``INSERT`` statement, the engine calls the ``beginInsert()``
+method in the connector, which receives a table handle and a list of columns.
+It should return a ``ConnectorInsertTableHandle``, that can carry any
+connector specific information and will be passed to the page sink provider.
```

Comment:
```suggestion
connector specific information, and it is passed to the page sink provider.
```

Reply:
I forgot about using the present tense, but I wanted to ask about passive voice - is `it is passed` passive and should we change it to say that the engine passes it?